client/src/api/user-api.js

```diff
@@ -0,0 +1,20 @@
+import api from "./api";
+
+async function createUser(auth0_Id, controller) {
+  return api(
+    {
+      method: "post",
+      endpoint: "/user/create",
+      body: {
+        auth0_Id: auth0_Id,
```

Comment:
do this instead
auth0_id: auth0Id

the server checks for "auth0_id"

Reply:
This'll cause conflicts otherwise?

Reply:
> This'll cause conflicts otherwise?

the server would read undefined and itll throw an error